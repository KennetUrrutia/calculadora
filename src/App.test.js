import { render, screen } from "@testing-library/react"
import App from "./App"

describe('Pruebas sobre el componente App', () => {
  const num1 = 10
  const num2 = 20

  test('Debe de hacer match con nel snaptshot', () => { 
    const { container } = render(
      <App />
    )

    expect(container).toMatchSnapshot()
  })

  test('Debe de inicializar la calculadora con el valor de 0', () => {

  })

  test('Debe de calcular el valor correcto de la suma de 2 numeros', () => {
    
  })
  
  test('Debe de calcular el valor correcto de la resta de 2 numeros', () => {

  })

  test('Debe de calcular el valor correcto de la multiplicaciÃ³n de 2 numeros', () => {

  })
})