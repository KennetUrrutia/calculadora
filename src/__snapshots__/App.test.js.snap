// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre el componente App Debe de hacer match con nel snaptshot 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="contenedor-calculadora"
    >
      <div
        class="input"
      >
        0
      </div>
      <div
        class="fila"
      >
        <div
          class="boton-contenedor"
        >
          1
        </div>
        <div
          class="boton-contenedor"
        >
          2
        </div>
        <div
          class="boton-contenedor"
        >
          3
        </div>
        <div
          class="boton-contenedor operador"
        >
          +
        </div>
      </div>
      <div
        class="fila"
      >
        <div
          class="boton-contenedor"
        >
          4
        </div>
        <div
          class="boton-contenedor"
        >
          5
        </div>
        <div
          class="boton-contenedor"
        >
          6
        </div>
        <div
          class="boton-contenedor operador"
        >
          -
        </div>
      </div>
      <div
        class="fila"
      >
        <div
          class="boton-contenedor"
        >
          7
        </div>
        <div
          class="boton-contenedor"
        >
          8
        </div>
        <div
          class="boton-contenedor"
        >
          9
        </div>
        <div
          class="boton-contenedor operador"
        >
          *
        </div>
      </div>
      <div
        class="fila"
      >
        <div
          class="boton-contenedor"
        >
          =
        </div>
        <div
          class="boton-contenedor"
        >
          0
        </div>
        <div
          class="boton-contenedor"
        >
          .
        </div>
        <div
          class="boton-contenedor operador"
        >
          /
        </div>
      </div>
      <div
        class="fila"
      >
        <div
          class="boton-clear"
        >
          Clear
        </div>
      </div>
    </div>
  </div>
</div>
`;
